instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/internal/classes/ClassesLoadedHandler.java

```diff
@@ -21,18 +21,18 @@
  */
 public final class ClassesLoadedHandler implements RecordedEventHandler {
   /**
-   * process.runtime.jvm.classes.loaded is the total number of classes loaded since JVM start. See:
+   * jvm.class.loaded is the total number of classes loaded since JVM start. See:
    * https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md
    */
-  private static final String METRIC_NAME_LOADED = "process.runtime.jvm.classes.loaded";
+  private static final String METRIC_NAME_LOADED = "jvm.class.loaded";
```

Comment:
I think this needs to be `classes` (plural):

```suggestion
  private static final String METRIC_NAME_LOADED = "jvm.class.loaded";
```

Reply:
Specification uses singular `class` https://opentelemetry.io/docs/specs/semconv/runtime/jvm-metrics/ so this does not need to changed.